.github/workflows/dependency-submission.yml

```diff
@@ -0,0 +1,25 @@
+# https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#gradle
```

Comment:
I found this link to be more helpful: https://github.com/marketplace/actions/build-with-gradle#the-dependency-submission-action